upload_files/README.md

```diff
@@ -43,5 +43,5 @@ Full documentation of this workflow can be found [here](https://api.sentera.com/
 ## Examples
 | Language | Run Command                       | Example Command |
 | :------- | :---------------------------------|-----------------|
-| Ruby     | `$ ruby file_upload.rb`           | `FIELDAGENT_ACCESS_TOKEN=AKgnGiLNpk1AjppMqWSWbiNMNUYepcP8EalLo_gYRpM FIELDAGENT_SERVER=https://api.sentera.com OWNER_TYPE=SURVEY OWNER_SENTERA_ID=9nfdf02_CO_ytwfAcmeOrg_CV_deve_c49295f37_230411_080321 ruby file_upload.rb` |
-| Ruby     | `$ ruby multipart_file_upload.rb` | `FIELDAGENT_ACCESS_TOKEN=AKgnGiLNpk1AjppMqWSWbiNMNUYepcP8EalLo_gYRpM FIELDAGENT_SERVER=https://api.sentera.com PARENT_SENTERA_ID=9nfdf02_CO_ytwfAcmeOrg_CV_deve_c49295f37_230411_080321 ruby multipart_file_upload.rb` |
+| Ruby     | `$ ruby file_upload.rb`           | `FIELDAGENT_ACCESS_TOKEN=AKgnGiLNpk1AjppMqWSWbiNMNUYepcP8EalLo_gYRpM FIELDAGENT_SERVER=https://api.sentera.com FILE_PATH="../test_files/test.geojson" CONTENT_TYPE="application/json" OWNER_TYPE=SURVEY OWNER_SENTERA_ID=9nfdf02_CO_ytwfAcmeOrg_CV_deve_c49295f37_230411_080321 ruby file_upload.rb` |
+| Ruby     | `$ ruby multipart_file_upload.rb` | `FIELDAGENT_ACCESS_TOKEN=AKgnGiLNpk1AjppMqWSWbiNMNUYepcP8EalLo_gYRpM FIELDAGENT_SERVER=https://api.sentera.com FILE_PATH="../test_files/test.tif" CONTENT_TYPE="image/tiff" PARENT_SENTERA_ID=9nfdf02_CO_ytwfAcmeOrg_CV_deve_c49295f37_230411_080321 ruby multipart_file_upload.rb` |
```

Comment:
Added `FILE_PATH` and `CONTENT_TYPE` environment variables to these two example commands so that it's clear what all the variables/inputs are for those examples.